webdev2.html

```diff
@@ -0,0 +1,94 @@
+<!DOCTYPE html>
+<html>
+<head>
+    <meta charset="utf-8">
+    <title>Leaflet Map</title>
+    <!-- External Stylesheets -->
+    <link rel="stylesheet" href="https://unpkg.com/leaflet@1.7.1/dist/leaflet.css" />
+    <script src="https://unpkg.com/leaflet@1.7.1/dist/leaflet.js"></script>
+    <style>
+        html,
+        body {
+            width: 100%;
+            min-height: 100%;
+        }
+
+        body {
+            padding: 0;
+            margin: 0;
+        }
+
+        #map {
+            top: 60px;
+        }
+
+        #header {
+            top: 0;
+            width: 100%;
+            z-index: 9999999;
+            position: fixed;
+            background-color: cadetblue;
+            color: aliceblue;
+            font-family: monospace;
+            padding-left: 20px;
+
+        }
+
+        #header * {
+            display: inline-block;
+            margin-right: 20px
+        }
+    </style>
+    // add omnivore
```

Comment:
Confusing as it is, this the wrong syntax for HTML comments.
Here's some guidance:
 - HTML `<!-- -->`
 - CSS `/* */`
 - JavaScript `//`
You need to be between the `<script>` tags in order to use a JavaScript comment.